content/docs/code-splitting.md

```diff
@@ -111,45 +111,96 @@ If you're setting up Webpack yourself, you'll probably want to read Webpack's
 
 **Before:**
 
 ```js
 import OtherComponent from './OtherComponent';
 
 const MyComponent = () => (
-  <OtherComponent/>
+  <div>
+    <OtherComponent/>
+  </div>
 );
 ```
 
 **After:**
 
 ```js
-import Loadable from 'react-loadable';
+const OtherComponent = React.lazy(() => import('./OtherComponent'));
 
-const LoadableOtherComponent = Loadable({
-  loader: () => import('./OtherComponent'),
-  loading: () => <div>Loading...</div>,
-});
+const MyComponent = () => (
+  <div>
+    <OtherComponent />
+  </div>
+);
+```
+
+This will automatically load the bundle containing the `OtherComponent` when this component gets rendered.
+
+`React.lazy` takes a function that must call a dynamic `import()`. This must return a `Promise` which resolves to a module with a `default` export containing a React component.
+
+### Suspense
+
+If the module containing the `OtherComponent` is not yet loaded by the time `MyComponent` renders, we must show some fallback content while we're waiting for it to load - such as a loading indicator. This is done using the `Suspense` component.
+
+```js
+const OtherComponent = React.lazy(() => import('./OtherComponent'));
 
 const MyComponent = () => (
-  <LoadableOtherComponent/>
+  <div>
+    <Suspense fallback={<div>Loading...</div>}>
+      <OtherComponent />
+    </Suspense>
+  </div>
 );
 ```
 
-React Loadable helps you create
-[loading states](https://github.com/thejameskyle/react-loadable#creating-a-great-loading-component),
-[error states](https://github.com/thejameskyle/react-loadable#loading-error-states),
-[timeouts](https://github.com/thejameskyle/react-loadable#timing-out-when-the-loader-is-taking-too-long),
-[preloading](https://github.com/thejameskyle/react-loadable#preloading), and
-more. It can even help you [server-side render](https://github.com/thejameskyle/react-loadable#------------server-side-rendering) an app with lots of code-splitting.
+The `fallback` prop accepts any React elements that you want to render while waiting for the component to load. You can place the `Suspense` component anywhere above the lazy component. You can even wrap multiple lazy components with a single `Suspense` component.
+
+```js
+const OtherComponent = React.lazy(() => import('./OtherComponent'));
+const AnotherComponent = React.lazy(() => import('./AnotherComponent'));
+
+const MyComponent = () => (
+  <div>
+    <Suspense fallback={<div>Loading...</div>}>
+      <section>
+        <OtherComponent />
+        <AnotherComponent />
+      </section>
+    </Suspense>
+  </div>
+);
+```
+
+### Error boundaries
+
+If the other module fails to load it will trigger an error. For example, due to network failure. To handle these errors to show a nice user experience and manage recovery see the [Error boundaries](/docs/error-boundaries.html) guide. Once you've created your Error boundary. You can use it anywhere above your lazy components to display an error state when there's a network error.
```

Comment:
Breaking up sentences like this is a bit hard to read to me.

Can you rephrase without breaking if ... for example ... to handle these?